src/js/service/game.js

```diff
@@ -1,36 +1,108 @@
 import { GAME_CONDITION } from '../constants/condition.js';
+import { SELECTOR } from '../constants/selector.js';
 
 import gameSetting from '../model/GameSetting.js';
 import Car from '../model/Car.js';
 
 import { generateRandomNumber } from '../utils/index.js';
+import { $ } from '../utils/dom.js';
+
+import {
+  hideGamePlay,
+  removeAllSpinners,
+  showCarNames,
+  showGamePlay,
+  showMoving,
+} from '../view/playGame.js';
+import { showWinners, showGameResult, hideGameResult } from '../view/gameResult.js';
+import {
+  enableCarNamesForm,
+  enableTrialCountForm,
+  hideTrialCountForm,
+  resetCarNamesForm,
+  resetTrialCountForm,
+} from '../view/gameSettingForm.js';
 
 const isMovable = () => {
-  const { MIN_RANDOM_NUMBER, MAX_RANDOM_NUMBER } = GAME_CONDITION;
+  const { MIN_RANDOM_NUMBER, MAX_RANDOM_NUMBER, MOVABLE_MIN_NUMBER } = GAME_CONDITION;
   const randomNumber = generateRandomNumber(MIN_RANDOM_NUMBER, MAX_RANDOM_NUMBER);
 
-  return randomNumber >= GAME_CONDITION.MOVABLE_MIN_NUMBER;
+  return randomNumber >= MOVABLE_MIN_NUMBER;
 };
 
-const updateCarDistance = (car) => {
+const updateCarMoving = (car) => {
   if (isMovable()) {
     car.move();
+    showMoving(car);
   }
 };
 
 const raceOneTurn = (cars) => {
-  cars.forEach((car) => updateCarDistance(car));
+  cars.forEach((car) => updateCarMoving(car));
+};
+
+const getWinners = (cars) => {
+  const distanceByCar = cars.map((car) => car.getDistance());
+  const maxDistance = Math.max(...distanceByCar);
```

Comment:
```suggestion
  const maxDistance = Math.max(...cars.map(car => car.getDistance());
```
이렇게 사용하고 winner는 cars에서 바로 filter + map으로 리턴하는게 보다 더 선언적으로 작성이 될 것 같아요
`cars[idx]`로 객체에 접근하는게 이해하기 어려울수도 있겠다고 느껴졌습니다~

Reply:
> cars[idx]로 객체에 접근하는게 이해하기 어려울수도 있겠다고 느껴졌습니다

다시 보니 직관적으로 이해하기 어려울 것 같습니다 👀
`forEach` + `if` 대신 `filter`를 사용함으로써 depth도 줄이고 가독성이 좋아졌습니다👍